module-2/backend/week_one.md

```diff
@@ -5,30 +5,84 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  get - retrieve a view
+  post - update a portion of the view with the given parameter
+  put - generate new data and return it in the view.
+  delete - delete information
+  
 2. What is Sinatra?
+  Sinatra is a framework language for web development.  My understanding is that it lays out the routes
+  and structure for all of the components to interact with each other.
+  
 4. What is MVC?
+  Model View Controller.  I'm uncertain if this refers to all of them as a concept or just the controller.  The controller's        purpose is to delineate the actions that are taken whenever a request is made.
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  We follow conventions for the sake of readability and understanding between developers.
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  The parameters of the request I think.
+  
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  I'm not sure how to pass a local variable.

```

Comment:
Syntax for a local variable below:

```ruby
get '/horses' do
  name = "Mr. Ed" # Assign "Mr. Ed" to a local variable.
  erb :index, locals: {name: name} # Pass it to our view.
end
```

Alternatively we can skip the assignment:

```ruby
get '/horses' do
  erb :index, locals: {name: "Mr. Ed"} # Passes "Mr. Ed" to the view as a local variable that we can use in the view with just `<%= name %>`.
end
```